src/Components/Button.js

```diff
@@ -32,6 +43,12 @@ const Button = (props) => {
 				<div className={classes.modalButtonText}>
 					{props.text}
 				</div>
+				<div style={{ position: 'absolute', right: '32px', color: '#8E9093', fontSize: '14px' }}>
+					{props.command &&
+						<span className={classes.modalCommandIcon}>⌥</span> 
+					}
+					<span className={classes.modalKeyIcon}>{props.shortcut}</span>
```

Comment:
For the styling for this, instead of applying the background and border radius directly to the text, i'd recommend putting the text in a div and then playing around with height/width and padding to make it look more like the designs. The background is currently much more narrow than in the designs.

Implementation: 
![image](https://user-images.githubusercontent.com/33686587/133519028-8af4f8ab-341f-4be9-ad80-67721cc5deff.png)

The design:
![image](https://user-images.githubusercontent.com/33686587/133519120-dc7b001f-f0b7-4517-bcf0-3ca9033673ea.png)